state/machine_ports_ops_test.go

```diff
@@ -0,0 +1,346 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package state
+
+import (
+	"github.com/juju/collections/set"
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/juju/core/network"
+)
+
+type MachinePortsOpsSuite struct {
+}
```

Comment:
it is useful even using the testing.IsolationSuite just in case someone adds some logging later.